src/main/java/mate/academy/spring/dao/impl/MovieSessionDaoImpl.java

```diff
@@ -57,4 +59,51 @@ public Optional<MovieSession> get(Long id) {
             throw new DataProcessingException("Can't get a movie session by id: " + id, e);
         }
     }
+
+    @Override
+    public MovieSession update(MovieSession movieSession) {
+        Session session = null;
+        Transaction transaction = null;
+        try {
+            session = sessionFactory.openSession();
+            transaction = session.beginTransaction();
+            session.update(movieSession);
+            transaction.commit();
+            return movieSession;
+        } catch (Exception e) {
+            if (transaction != null) {
+                transaction.rollback();
+            }
+            throw new DataProcessingException("Can't update a movie session: " + movieSession, e);
+        } finally {
+            if (session != null) {
+                session.close();
+            }
+        }
+    }
+
+    @Override
+    public boolean delete(Long id) {
+        Session session = null;
+        Transaction transaction = null;
+        try {
+            session = sessionFactory.openSession();
+            transaction = session.beginTransaction();
+            Query<MovieSession> query = session.createQuery("DELETE FROM MovieSession "
+                    + "WHERE id = :id", MovieSession.class);
```

Comment:
why MovieSession.class parameter needed in delete query?
```suggestion
 Query query = session.createQuery("DELETE FROM MovieSession  WHERE id = :id");
```